### PR TITLE
Add another services

### PR DESCRIPTION
**Pull request by mladjan-gadzic**

### CODE REVIEW

File: src/main/java/com/example/collageapp/controller/CourseController.java [hunk 1]
```diff
@@ -1,48 +1,42 @@
...
 import com.example.collageapp.entity.Course;
-import com.example.collageapp.repository.CourseRepository;
+import com.example.collageapp.service.CourseService;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
 @RestController
 @RequestMapping("/api/courses")
 public class CourseController {
```
Comment (mladjan-gadzic) [inline src/main/java/com/example/collageapp/controller/CourseController.java:11]: Create POST mapping as well

File: src/main/java/com/example/collageapp/controller/StudentController.java [hunk 2]
```diff
@@ -10,39 +10,39 @@
 @RequestMapping("/api/students")
```
Comment (mladjan-gadzic) [inline src/main/java/com/example/collageapp/controller/StudentController.java:11]: Remove this newline

File: src/main/java/com/example/collageapp/controller/CourseController.java (hunk 1 above)
Comment (KacaS97) [reply src/main/java/com/example/collageapp/controller/CourseController.java:11]: Done

File: src/main/java/com/example/collageapp/controller/StudentController.java (hunk 2 above)
Comment (KacaS97) [reply src/main/java/com/example/collageapp/controller/StudentController.java]: Done

**Pull request merged**